src/components/VideoCard/VideoCard.js

```diff
@@ -1,23 +1,42 @@
 import React, { useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
-import { useAuth, useWatchLater } from "../../context";
+import { useAuth, useWatchLater, useHistory } from "../../context";
 import { addToWatchLater } from "../../services/addToWatchLater";
+import { addToHistory } from "../../services/addToHistory";
```

Comment:
can add an index file in services since there are many services can directly import from index